core/src/test/resources/org/fao/geonet/api/Messages.properties

```
@@ -77,6 +81,21 @@ register_email_message=Dear User,\n\
   \n\
   Yours sincerely,\n\
   The %s team.
+register_email_group_message=Dear User,\n\
+  Your registration at %s was successful.\n\
+  Your account is: \n\
+  * username: %s\n\
+  * password: %s\n\
+  * profile: %s\n\
+  \n\
+  You've told us that you want to be %s in group %s, you will be contacted soon.\n\
```

```suggestion
 We have sent your request for %s to the group %s administrator. You will be contacted shortly.\n\
```

I would prefer to change the sentence around to "We have sent your request to the group %s administrator. They will be in contact about %s access." to be clear that it will be the group administrator contact them (and not the application).